systems.csv

```diff
@@ -22,11 +22,15 @@ BR,Bike Itaú - Riviera,"Riviera, BR",rivi_bike,https://www.rivieradesaolourenco
 BR,Bike VV,"Vila Velha, BR",bike_vv,https://www.bikevv.com.br/,https://vilavelha.publicbikesystem.net/ube/gbfs/v1/
 CA,Bike Share Toronto,"Toronto, ON",bike_share_toronto,https://www.bikesharetoronto.com/,https://tor.publicbikesystem.net/ube/gbfs/v1/
 CA,BIXI-Montreal,"Montreal, QC",Bixi_MTL,https://www.bixi.com/,https://api-core.bixi.com/gbfs/gbfs.json
+CA,Bunny,"Kelowna, BC",bunny_kelowna,http://bunnyscooters.com/,https://bunny-api.joyridecity.bike//gbfs/en/gbfs.json
 CA,HOPR Ottawa,"Ottawa, ON",6,https://gohopr.com/velogo/,https://gbfs.hopr.city/api/gbfs/6/
 CA,HOPR Vancouver,"Vancouver, BC",13,https://www.mobibikes.ca/,https://gbfs.hopr.city/api/gbfs/13/
 CA,Mobi Bike Share,"Vancouver, BC",mobi_bikes,https://www.mobibikes.ca/,https://vancouver-gbfs.smoove.pro/gbfs/gbfs.json
 CA,nextbike Canada,"Victoria, CA",nextbike_ca,https://canada.nextbike.net/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_ca/gbfs.json
+CA,ROLL Technologies Inc,"Kelowna, BC",roll_kelowna,https://www.rollscooters.com/,https://main.rollapi.com/v1/gbfs/gbfs.json
```

Comment:
```suggestion
CA,ROLL Technologies Inc,"Kelowna, BC",0,https://www.rollscooters.com/,https://main.rollapi.com/v1/gbfs/gbfs.json
```
Initially I had asked for this to be revised, but see comment in thread about needing to clean-up ids later.

Reply:
It also appears that this API is throwing errors on at least a few of its pages sporadically
![Screen Shot 2020-02-17 at 12 58 39 AM](https://user-images.githubusercontent.com/2909/74627425-dacea300-5120-11ea-9029-c51c0a8953a0.png)
.

Reply:
Hi, @jcn and @heidiguenin . This is Roll Technologies Inc. Our GBFS apis are recently changed to under 

> https://data.rollapi.com/

Sorry for your inconvenience.